tests/scripts/check-test-cases.py

```diff
@@ -20,26 +20,30 @@
 #
 # This file is part of Mbed TLS (https://tls.mbed.org)
 
+import argparse
```

Comment:
Why are the changes needed in this file/commit needed in this PR?

Reply:
The new option to `check-test-cases` isn't needed to make `psa_hash_compute` work. But I made it and used it during the development of the PR to make it easier to see errors (they tend to get lost among the warnings).